content/post/yet-another-reason-to-privatize-the-police.md

```diff
@@ -34,21 +34,21 @@ This brings us to the second major problem with the state-funded police: the pro
+But rather than subjecting state-funded police to the same mechanisms that make market economies so competitive and flowing with choice &mdash; freedom of entry, the profit and loss mechanism, competitive innovation, and the desire to reduce business costs &mdash; there is no question that a majority of our nation would rather it be the sacrosanct duty of the state alone to provide us with policing services. And why should that be? Policing services are no more unique than any other service, nor is their provision too complicated for the world's most technologically advanced and innovative private sector[^4]. Why have a one-size-fits-all, forcibly funded monopoly on police provision as opposed to tens, hundreds, or thousands of voluntarily funded, private policing firms that are actually beholden to their customers and tied to the profit and loss system of the marketplace?
 
-Businesses, apartment complexes, neighborhoods, and other key stakeholders in a community's well-being are strongly incentivized to provide policing services &mdash; independently or collectively &mdash; so that consumers can safely patronize businesses and individuals can feel safe living in their apartments and neighborhoods. And is there any objection to the obvious fact that the incentive for community stakeholders to offer policing services would be met with an equal demand for security on behalf of individual consumers, property owners, and business owners, none of whom desire to live in unsafe neighborhoods or conduct businesses in dangerous communities? So long as consumer demand exists for safety and protection, which it always will, free market entrepreneurship will direct the necessary resources to meet those demands.[^5]
+Businesses, apartment complexes, neighborhoods, and other key stakeholders in a community's well-being are strongly incentivized to provide policing services &mdash; independently or collectively &mdash; so that consumers can safely patronize businesses and individuals can feel safe living in their apartments and neighborhoods. Should we not expect that the incentive for community stakeholders to offer policing services would also be met with an equal demand for security on behalf of individual consumers, property owners, and business owners, none of whom desire to live in unsafe neighborhoods or conduct businesses in dangerous communities? So long as consumer demand exists for safety and protection, which it always will, free market entrepreneurship will direct the necessary resources to meet those demands.[^5]
```

Comment:
```suggestion
Businesses, apartment complexes, neighborhoods, and other key stakeholders in a community's well-being are strongly incentivized to provide policing services &mdash; independently or collectively &mdash; so that consumers can safely patronize businesses and individuals can feel safe living in their apartments and neighborhoods. Should we not expect that the incentive for community stakeholders to offer policing services would also be met with an equal demand for security on behalf of individual consumers, property owners, and business owners, none of whom desire to live in unsafe neighborhoods or conduct businesses in dangerous communities? So long as consumer demand exists for safety and protection, a condition likely to persist this side of heaven, free market entrepreneurship will direct the necessary resources to meet those demands.[^5]
```